docs/future_notes.md

```diff
@@ -0,0 +1,29 @@
+# Some ideas on supporting different chrome/java versions
+
+In trying this in trove it didn't work great because of the differences in ARM and x64 macs.
+
+## alternative java installation
+```
+RUN wget -qO - https://packages.adoptium.net/artifactory/api/gpg/key/public| apt-key add - \
+    && apt-get update \
+    && apt-get install -y temurin-8-jdk
+
+RUN update-java-alternatives --set /usr/lib/jvm/temurin-8-jdk-amd64
+
+RUN apt-get update && apt-get install -y wget unzip libatk1.0-0 libgtk-3-0 libgbm1
+```
+## Download and extract Google Chrome
+
+attempts using thorium
+
+```
+RUN wget https://github.com/Alex313031/thorium/releases/download/M114.0.5735.134/thorium-browser_114.0.5735.134_amd64.zip && \
+    unzip chrome-linux.zip -d /usr/local/bin/ && \
+    rm chrome-linux.zip
+```
```

Comment:
I am getting an error. I can't resolve from this. 
``` 
failed to solve: executor failed running [/bin/sh -c wget https://github.com/Alex313031/thorium/releases/download/M114.0.5735.134/thorium-browser_114.0.5735.134_amd64.zip      unzip chrome-linux.zip -d /usr/local/bin/]: exit code: 4 
```

Reply:
Huh? That change isn't in a file, its just in markdown

Reply:
Did you pull updates?